make.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env -S uv run
 # SPDX-License-Identifier: MIT OR Apache-2.0
 # SPDX-FileCopyrightText: The Ferrocene Developers
+# SPDX-FileCopyrightText: The Rust Project Contributors
 
 import os
-import ferrocene_standalone_make_cli
+from src import ferrocene_standalone_make_cli
```

Comment:
Instead of copying the `ferrocene_standalone_make_cli` module into the repo and importing it, we should just replace `make.py` with the contents of that file, and adding a call to the `main` function at the end of it.

Originally this was a module inside of ferrocene-sphinx-shared-resources as we wanted to share the `make.py` implementation across multiple Sphinx sites, but we don't need that once we copy the file inside of this repo.